rotkehlchen/tasks/manager.py

```diff
@@ -711,6 +716,22 @@ def _maybe_detect_new_spam_tokens(self) -> Optional[list[gevent.Greenlet]]:
             user_db=self.database,
         )]
 
+    def _maybe_detect_new_spam_tokens_augmented(self) -> Optional[list[gevent.Greenlet]]:
+        """
+        This function queries the globaldb looking for assets that look like spam tokens
+        and ignores them in addition to marking them as spam tokens
```

Comment:
This is copy pasta from the other task ...

```suggestion
        This function runs the augmented token detection algorithm which is a heavier and more
        time consuming analysis on user's asset that involves external calls in order to find and detect
        potential spam tokens.
```